### PR TITLE
new sfs output for global-workflow

### LINKED ISSUE #1067: SFS Output Requested for Baseline Runs
The baseline runs of SFS are being created and users requested SSRUN. In addition, minor changes are needed to run with g-w.

### PR DESCRIPTION
**Pull request by NeilBarton-NOAA**
add SSRUN variable for SFS output and file for g-w scripts as requested by developers 

PR in response to https://github.com/NOAA-EMC/UPP/issues/1067

### CODE REVIEW

**Comment by WenMeng-NOAA** [comment]
@NeilBarton-NOAA Could you open a new [UPP issue](https://github.com/NOAA-EMC/UPP/issues) to link your PR?

**Comment by NeilBarton-NOAA** [comment]
@WenMeng-NOAA Issue added https://github.com/NOAA-EMC/UPP/issues/1067

File: parm/sfs/postcntrl_sfs.xml [hunk 1]
```diff
@@ -657,21 +663,6 @@
        <scale>6.0</scale>
     </param>
 
-    <param>
```
Comment (WenMeng-NOAA) [inline parm/sfs/postcntrl_sfs.xml]: @NeilBarton-NOAA The following three max wind products are requested by CPC. Please keep them.

File: parm/sfs/postcntrl_sfs.xml [hunk 2]
```diff
@@ -299,6 +299,12 @@
        <table_info>NCEP</table_info>
        <scale>3.0</scale>
     </param>
+    
+    <param>
```
Comment (WenMeng-NOAA) [inline parm/sfs/postcntrl_sfs.xml:303]: @NeilBarton-NOAA SSRUN is already in the SFS UPP control file. Please confirm that atmosphere top downward shortwave is needed.

Comment (NeilBarton-NOAA) [reply parm/sfs/postcntrl_sfs.xml:303]: yes, I was confused with the gefs xml. AVE_DSWRF_ON_TOP_OF_ATMOS is the added value

**Comment by NeilBarton-NOAA** [comment]
PR is update that includes the CPC requested variables and adding AVE_DSWRF_ON_TOP_OF_ATMOS

**Comment by WenMeng-NOAA** [comment]
@NeilBarton-NOAA I have the UPP standalone test for SFS at /u/wen.meng/ptmp/post_sfs_2022090100 on Cactus. The new variable added in your PR was generated as follows:
```
wgrib2 GFSPRS.GrbF12 -match DSWRF
369:21296232:d=2022090100:DSWRF:top of atmosphere:6-12 hour ave fcst:
```
Please take a look my test and let me know if you see issues.

**Comment by NeilBarton-NOAA** [comment]
@WenMeng-NOAA I'm having issue connecting to catcus now. Could you transfer to hera? Thanks

**Comment by WenMeng-NOAA** [comment]
@NeilBarton-NOAA Please see /home/Wen.Meng/stmp2/post_sfs_2022090100 on Hera.

**Comment by NeilBarton-NOAA** [comment]
Thank you @WenMeng-NOAA . The output looks good to me

**Comment by WenMeng-NOAA** [comment]
@gspetro-NOAA @FernandoAndrade-NOAA You may start the UPP RTs on R&D machines. There should be no baseline changes with this PR since SFS test is not included in the UPP RTs.

**Comment by WenMeng-NOAA** [comment]
The UPP RTs have been completed on WCOSS2 with no baseline changes.

**Review by gspetro-NOAA — APPROVED** [review]
Tests pass on Hera, Orion, and Hercules w/no baseline changes.

**Review by WenMeng-NOAA — APPROVED** [review]

**Comment by WenMeng-NOAA** [comment]
This PR is ready for merging.

**Pull request merged**